test/nnp_fusion.cpp

```diff
@@ -43,7 +43,7 @@
 using namespace ngraph;
 using namespace std;
 
-TEST(Argon_fusion, fuse_max_with_constant_zero_input_as_relu)
+TEST(nnp_fusion, fuse_max_with_constant_zero_input_as_relu)
```

Comment:
Should this be moved to the nnp repo?

Reply:
I need to move this to backendtest

Reply:
removed the tests as it was taken care in the `CoreFusion` PR